bundles/config/org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs.test/src/test/java/org/eclipse/smarthome/config/discovery/usbserial/linuxsysfs/internal/SysFsUsbSerialScannerTest.java

```diff
@@ -153,6 +153,32 @@ public void testNonUsbDeviceIsSkipped() throws IOException {
         assertThat(scanner.scan(), is(empty()));
     }
 
+    @Test
+    public void testCanPerformScans() {
+        // with given test setup, scans can be performed
+        assertThat(scanner.canPerformScans(), is(true));
```

Comment:
For simple true/false checks, can you please change it to `assertTrue`/`assertFalse`? Saves you a couple of characters and makes it more readable :) This also applies to the other occurrences.

Reply:
I disagree that the suggested change would render the code more readable, because I think that the phrase "assert that x is true" sounds more natural than "assert true x". I also think that this is question of personal preference, so I probably cannot convince you that my preference is the better choice :)

Is there some coding guideline regarding this for ESH? I just found the following on the [ESH testing page](https://www.eclipse.org/smarthome/documentation/development/testing.html) (which suggests the solution I chose):

> Using the the hamcrest matcher library is a good way to write expressive assertions. In contrast to the original assertion statements from JUnit the hamcrest matcher library allows to define the assertion in a more natural order

Should I still change this?

Reply:
Personal preference, leave it as it is :)